examples/development/docker-compose-dev.yml

```diff
@@ -156,6 +177,28 @@ services:
       FULL_TEXT_INDEX: ${FULL_TEXT_INDEX}
       CUSTOM_INDEX: ${CUSTOM_INDEX}
       AUTOCOMPLETE_INDEX: ${AUTOCOMPLETE_INDEX}
+      FILE_INDEX: ${FILE_INDEX}
+
+
+  about:
+    container_name: tr_about
+    image: knawhuc/textrepo-about:${DOCKER_TAG}
+    build:
+      context: ./about
+      args:
+        SOURCE_COMMIT: ${SOURCE_COMMIT}
+        DOCKER_TAG: ${DOCKER_TAG}
+    networks:
+      - textrepo_network
+    volumes:
+      - aboutdata:/about
+    environment:
+      ABOUT_TR_DESCRIPTION: ${ABOUT_TR_DESCRIPTION}
+      ABOUT_TR_DOCUMENTATION: ${ABOUT_TR_DOCUMENTATION}
+      ABOUT_TR_SOURCECODE: ${ABOUT_TR_SOURCECODE}
+      NGINX_VERTUAL_HOST_SCHEME: ${NGINX_VERTUAL_HOST_SCHEME}
```

Comment:
Same typo as in docker-compose.env, which is why it still works :-)

Reply:
6daab35